src/Symfony/Component/Uid/BinaryUtil.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Uid;
+
+/**
+ * @internal
+ *
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+class BinaryUtil
```

Comment:
Even if it's a utility class, should we add explicit tests for it?

Reply:
It's already tested since we use the code in Uuid and Ulid.
If someone whishes to contribute more specific tests, PR welcome.